src/inspector/network_resource_manager.h

```diff
@@ -0,0 +1,34 @@
+// network_resource_manager.h
+#ifndef SRC_INSPECTOR_NETWORK_RESOURCE_MANAGER_H_
+#define SRC_INSPECTOR_NETWORK_RESOURCE_MANAGER_H_
+
+#include <atomic>
+#include <string>
+#include <unordered_map>
+
+namespace node {
+namespace inspector {
+
+class NetworkResourceManager {
+ public:
+  static void Put(const std::string& url, const std::string& data);
```

Comment:
The `put` can be invoked from `worker_threads` as well. This should avoid to be a static instance to be thread-safe.